zappa/core.py

```diff
@@ -2183,11 +2183,13 @@ def update_cognito(self, lambda_name, user_pool, lambda_configs, lambda_arn):
                 description_kwargs[key] = value
             elif key == "LambdaConfig":
                 for lckey, lcvalue in value.items():
+                    # update needed config
                     if lckey in LambdaConfig:
                         value[lckey] = LambdaConfig[lckey]
                 print("value", value)
                 description_kwargs[key] = value
```

Comment:
@sridhar562345 When associated with User Pool, Lambda Config is {}, so it will never add a user-defined configuration.